### DISCUSSION TITLE
formily2.0版本Field无法响应onBlur和onFocus事件？

### DISCUSSION THREAD

**Discussion by wittech** [alibaba/formily#1154] [Q&A 2.x] [answered]
测试了一下发现最新的2.0版本，表单input组件无法响应onFocus和onBlur事件，从field对象拿到的字段状态active一直是false。看了ReactiveField的实现代码，发现react的版本里面只有处理onChange事件，没有onFocus和onBlur的处理逻辑。而vue版本里面却是实现了的，请问是内核还没有完全实现？

**Comment by janryWang** [comment, marked as answer]
目前已修复了

**Reply by amilaxyz** [reply]
I am sorry can you please describe how to use onBlur with x-reactions
